Gemfile.lock

```diff
@@ -54,6 +55,10 @@ GEM
       unicode-display_width (~> 1.0, >= 1.0.1)
     ruby-progressbar (1.8.1)
     sass (3.4.22)
```

Comment:
I can see arguments either way - if it's a security-related change, not sure it should be up to the app to opt-in, but in the other case, it makes more sense if we provide the ability to do both before we force app developers to do so eventually.